### DISCUSSION TITLE
Why did the basic Unified hosts list suddenly jump by nearly 40k total hosts in the last update?

### DISCUSSION THREAD

**Discussion by MultiKoopa** [StevenBlack/hosts#2010] [Q&A] [answered]
it was at around 207k total hosts at the beginning of July, and the update from 2 days ago put it at about 244k total hosts now. What exactly happened in the last 2 weeks??

**Comment by StevenBlack** [comment, marked as answer]
Hi @MultiKoopa where do you get the numbers "*207k total hosts*" and "*about 244k total hosts*"? That's nowhere near the number of hosts we list.

That said... we very recently we grew by a large number of domains, yes. 

Most of the recent increase (35,000 domains) (between release 3.10.14 and 3.10.15) come via the Adguard-CNAME list. They've identified domains in the `zooplus.nl` and `zooplus.com` space, and others. As a result their list size has increased from about 27k domains to over 57k domains.

We often get spikes like this, and typically those spikes reduce fairly quickly thereafter. It varies by curator but typically it appears they blanket a provider and, thereafter, whitelist from there. This is evidently done in large batches, so we see swings of several thousand domains from one release to another.

Curating the internet is hard work 😄 and most of our list providers are mostly very diligent about it.

**Reply by MultiKoopa** [reply]
Oh I see. The default HostsMan action is to merge hosts files when updating instead just replacing. That would explain a lot. Thank you for helping me find this!

**Comment by StevenBlack** [comment]
@MultiKoopa this is interesting:  See Issue #2014 


<img width="616" alt="base_hosts_file_size" src="https://user-images.githubusercontent.com/80144/179655626-f71be279-51a0-4756-92a1-07cf18e903d6.png">
